tests/gis_tests/geoapp/tests.py

```diff
@@ -923,3 +925,15 @@ def test_non_concrete_field(self):
     def test_values_srid(self):
         for c, v in zip(City.objects.all(), City.objects.values()):
             self.assertEqual(c.point.srid, v['point'].srid)
+
+    def test_geometry_value_annotation(self):
```

Comment:
Is there a better place for these tests than the deprecated `GeoQuerySetTest` class?